packages/browser/src/BacktraceBrowserRequestHandler.ts

```diff
@@ -44,13 +44,14 @@ export class BacktraceBrowserRequestHandler implements BacktraceRequestHandler {
     ): Promise<BacktraceReportSubmissionResult<T>> {
         const controller = new AbortController();
         const id = setTimeout(() => controller.abort(), this._timeout);
+        const [signal, disposeSignal] = anySignal(abortSignal, controller.signal);
```

Comment:
I just notice - if someone pas a custom abort signal to the request handler, the setTimeout function will not work, right?

Reply:
why not? that's the whole reason for `anySignal`, to abort when any signal aborts

Reply:
you're correct - i forgot what anysignal does.